src/components/motor_output_reordering/MotorOutputReorderingCanvas.js

```diff
@@ -0,0 +1,269 @@
+'use strict';
```

Comment:
While I like the new organisation into folders by components a lot more than the existing organisation by file type, I am not sure of the approach to use snake_case for directories and PascalCase for files. What is the reasoning behind it?

Reply:
@mikeller  i wasn't able to find coding style about that so was trying to follow what already exists. Some folders I was are snake_case and and most of classes files are PascalCases in the existing code as far as I can see.

Reply:
Renamed files and folders to PascalCase